en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+# Choreo Expression Syntax
+
+Choreo expressions comprise a powerful expression syntax based on the Ballerina language for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you the common expressions that you can use while developing Choreo applications.
+
+## Supported Literals 
+
+### String
+
+A string is simply a sequence of characters. It is required to use double quotes (")  to mark the boundary between the starting and ending characters.
+
+#### Examples
+
+- ```
+  "Colombo"
+  ```
+- ```
+  "12345"
+  ```
+- ```
+  "Ballerina is a programming language intended for network-distributed applications."
+  ```
```

Comment:
Since we don't have any explanatory text with each example, shall we remove bullet points as it looks bit lengthy and unnecessary

Reply:
Removed from where applicable